meta-balena-common/recipes-containers/balena-supervisor/balena-supervisor/update-balena-supervisor.service

```diff
@@ -4,13 +4,10 @@ Requires=\
     balena.service \
     bind-etc-balena-supervisor.service \
     tmp.mount
-Wants=\
-    mnt-boot.mount \
-    balena-supervisor.service
+Wants=balena-supervisor.service
 After=\
     balena.service \
     tmp.mount \
-    mnt-boot.mount \
```

Comment:
Is this intentionally removed without replacement?

Reply:
@mtoman My understanding is that mount units with a corresponding `.automount` should automatically mount their required filesystems on usage. For example, `/mnt/sysroot/inactive` is not mounted on boot and isn't used by any service, but accessing that directory at runtime will mount the partition.

I suspect that we can safely remove _many_ explicit dependencies on mounts, though some cases may require explicit configuration (such as when a unit contains `DefaultDependencies=no`, or there's a dependency loop). However, this remains to be demonstrated by our automated testing.